mixer/pkg/server/server_test.go

```diff
@@ -123,24 +136,28 @@ func newTestServer(globalCfg, serviceCfg string) (*Server, error) {
 }
 
 func TestBasic(t *testing.T) {
-	s, err := newTestServer(globalCfg, serviceCfg)
-	if err != nil {
-		t.Fatalf("Unable to create server: %v", err)
-	}
+	for i := 0; i < 2; i++ {
```

Comment:
nit: simplify / make this consistent with other tests...

can we just make this something like:
```
runtimes := []struct{
   name string
   useNewRutime bool
}{
  { "old runtime", false},
  { "new runtime", true},
}

for _, runtime := range runtimes {
   t.Run(runtime.Name, func(t *testing.T) {
       s, err := newTestServer(..., ..., runtime.useNewRuntime)
       ...
   }
}

```

Reply:
Now really, replacing 2 lines of trivial to understand code with what you show is by no means "simplifying" anything. Since you wrote it, I'll paste it in there, but this doesn't rank very high on my list of most valuable code patterns...